config/routes.rb

```diff
@@ -453,8 +453,16 @@
     end
     resources :requests, controller: "challenge_requests"
     # challenge types
-    resource :gift_exchange, controller: 'challenge/gift_exchange'
-    resource :prompt_meme, controller: 'challenge/prompt_meme'
+    resource :gift_exchange, controller: 'challenge/gift_exchange' do
```

Comment:
Could you switch to double quotes  as long as you're here? (Just on these lines; no need to worry about the rest of the file.)